nym-vpn-core/crates/nym-vpnd/src/cli.rs

```diff
@@ -30,33 +29,38 @@ pub(crate) struct CliArgs {
     #[arg(long)]
     pub(crate) disable_service: bool,
 
-    #[cfg(windows)]
     #[command(flatten)]
     pub(crate) command: Command,
 }
 
-#[cfg(windows)]
 #[derive(Args, Debug, Clone)]
 #[group(multiple = false)]
 pub(crate) struct Command {
+    #[cfg(windows)]
     #[arg(long)]
     pub(crate) install: bool,
 
+    #[cfg(windows)]
     #[arg(long)]
     pub(crate) uninstall: bool,
 
     #[arg(long)]
     pub(crate) start: bool,
```

Comment:
On windows start here means send an event to start the background service already registered and handled by windows. Is it the same here? Otherwise I would suggest use the other flag

Reply:
Yeah I don't quite understand what start means, maybe we should document this command?